src/packagedcode/maven.py

```diff
@@ -1026,6 +1049,20 @@ def parse(location=None, text=None, check_is_pom=True, extra_properties=None):
     return package
 
 
+def get_maven_licenses(licenses):
```

Comment:
See my comment above. I think this function is not needed at all.

Reply:
Yep, I agree. We already have pom.licenses which is a list of ordered dict.